### ISSUE TITLE
[FVT]2.12:install xcat failed for xcatconfig compilation errors

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1004]
env:
all platform
build:latest
How to reproduce:
```
Installing: xCAT-server-4:2.12-snap201604280930 [..done]
Installing: xCAT-2.12-snap20160428091 [.done]
Additional rpm output:
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 953.
Global symbol "$outref" requires explicit package name at /opt/xcat/sbin/xcatconfig line 955.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 955.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 962.
Global symbol "$outref" requires explicit package name at /opt/xcat/sbin/xcatconfig line 964.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 964.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 972.
Global symbol "$outref" requires explicit package name at /opt/xcat/sbin/xcatconfig line 974.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 974.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 983.
Global symbol "$outref" requires explicit package name at /opt/xcat/sbin/xcatconfig line 985.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 985.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 988.
Global symbol "$cmd" requires explicit package name at /opt/xcat/sbin/xcatconfig line 992.
Execution of /opt/xcat/sbin/xcatconfig aborted due to compilation errors.

```

**Comment by daniceexi** [comment]
It caused by the removing of lines which include variables definition in the commit 850051bd75511b42e7373f85274b0f1dc92447b2. I fixed it by re-declare the variables.

**Comment by daniceexi** [comment]
The fix has been checked into 2.12 branch by commit 8bdfc11977a18713e02022ae73567731860c4406

**Comment by junxiawang** [comment]
verified
